posts/a-guinea-pig-in-the-cloud.md

```diff
@@ -0,0 +1,389 @@
+
+---
+authors:
+- Bertrand PINEL
+tags:
+- DevOps
+- Cloud
+- Serverless
+- Architectures
+- AWS
+- Ember
+- Front
+date: 2019-02-27T17:27:07.000Z
+title: "A Guinea Pig in the Cloud"
+image: https://github.com/bpinel/blog-usa/blob/master/images/2019/02/guinea-pigs.png
+---
+
+# Introduction
+
+This article is not about the latest performance of the Chinese space industry and it will not hurt any pets. It is just a teaser about an idea that popped in my mind a few months ago and is the basis for a small tool I have been writing since that moment. 
+
+The idea is very simple. On one hand, thanks to the Cloud Providers, it is now very easy to host both a web site and some REST services and to benefit from managed services like Databases. On the other hand, JavaScript frameworks like [Angular](https://angular.io/), [React](https://reactjs.org/) or [Ember](https://emberjs.com/)... 
+
+![Ember logo](https://www.emberjs.com/images/brand/ember_Tomster-Lockup-3a0f7b78.png)
```

Comment:
Looks like this link is failing. You could try this:
https://emberjs.com/images/tomster-twitter-card-c38bcf1af22002906470c99fed09c044.png

![Ember logo](https://emberjs.com/images/tomster-twitter-card-c38bcf1af22002906470c99fed09c044.png)

Reply:
Other than that, the article looks great! I'm going to try this out when I get the chance

Reply:
Thanks for your comment and suggestion. Hope you will find time to try my little tool.
I have updated the image logo according to your suggestion !
Thanks again.